lib/active_admin_import/model.rb

```diff
@@ -101,10 +101,12 @@ def csv_allowed_types
       [
           'text/csv',
           'text/x-csv',
+          'text/x-comma-separated-values',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.